client/my-sites/customer-home/cards/actions/quick-links-compact/index.jsx

```diff
@@ -0,0 +1,309 @@
+/**
```

Comment:
This is basically copied from the [wide version of the quick links card](https://github.com/Automattic/wp-calypso/blob/5fb0b042c067a32feaa9a4f00d8b7e90969cc95b/client/my-sites/customer-home/cards/actions/quick-links/index.jsx) but always returning a `FoldableCard`.

Reply:
Noice, totally fine to copy paste here. We can always delete the other card later if we don't end up diverging/need it.